qr-code/src/test/java/com/adyen/checkout/qrcode/DefaultQRCodeDelegateTest.kt

```diff
@@ -198,7 +210,23 @@ internal class DefaultQRCodeDelegateTest(
                     Activity(),
                 )
 
-                assertEquals(awaitItem(), QrCodeComponentViewType.QR_CODE)
+                assertEquals(awaitItem(), QrCodeComponentViewType.SIMPLE_QR_CODE)
```

Comment:
The order of the arguments here is reversed, should be `assertEquals(expected, actual)`